osm2streets/src/output.rs

```diff
@@ -161,6 +165,33 @@ impl StreetNetwork {
                 }
             }
 
+            // Lanes through intersections
+            for inter in self.intersections.values() {
+                for Turn { movements, .. } in &inter.turns {
+                    let Some(movements) = movements else { continue };
```

Comment:
OK I actually hadn't understood the point of let-else till this; very cool!

Reply:
Yep, a bunch of nesting can be replaced with early returns! The `else` block can run any code, as long as it diverges (doesn't run to the end).